scrunch/expressions.py

```diff
@@ -526,7 +526,7 @@ def category_ids(var_id, var_value, variables=variables):
         if len(subitems) == 2:
             if 'value' in subitems[1] and 'variable' in subitems[0]:
                 var_url = subitems[0]['variable']
-                if ds.variables.index[var_url]['type'] == 'multiple_response':
+                if var_url in ds.variables.index and ds.variables.index[var_url]['type'] == 'multiple_response':
```

Comment:
I will make a call just once, but I cannot use the value of `get_dataset_variables` because that  is a dictionary keyed by variable alias, I would have to iterate over items of the dict and match the one that has the variable url. It was like that in the past, and you commented and said that was scary too, because iterating a potential huge dictionary doesn't work either. So I will stick with callen `ds.variables.index` just once